File: unittest/scripts/auto/js_devapi/scripts/mysqlx_tabledelete_help.js
//@ Initialization
shell.connect(__uripwd + '/mysql');
var table = db.user;
var crud = table.delete();
session.close();

//@ TableDelete help
crud.help();

//@ TableDelete help, \? [USE:TableDelete help]
\? TableDelete

//@ Help on bind
crud.help('bind');

//@ Help on bind, \? [USE:Help on bind]
\? TableDelete.bind

//@ Help on delete
crud.help('delete');

//@ Help on delete, \? [USE:Help on delete]
\? TableDelete.delete

//@ Help on execute
crud.help('execute');

//@ Help on execute, \? [USE:Help on execute]
\? TableDelete.execute

//@ Help on help
crud.help('help');

//@ Help on help, \? [USE:Help on help]
\? TableDelete.help

//@ Help on limit
crud.help('limit');

//@ Help on limit, \? [USE:Help on limit]
\? TableDelete.limit

//@ Help on orderBy
crud.help('orderBy');

//@ Help on orderBy, \? [USE:Help on orderBy]
\? TableDelete.orderBy

//@ Help on where
crud.help('where');

//@ Help on where, \? [USE:Help on where]
\? TableDelete.where

File: unittest/scripts/auto/py_devapi/scripts/mysqlx_tabledelete_help.py
shell.connect(__uripwd + '/mysql')
tabledelete = db.user.delete()
session.close()

#@ tabledelete
tabledelete.help()

#@ global ? for TableDelete[USE:tabledelete]
\? TableDelete

#@ global help for TableDelete[USE:tabledelete]
\help TableDelete

#@ tabledelete.execute
tabledelete.help('execute')

#@ global ? for execute[USE:tabledelete.execute]
\? TableDelete.execute

#@ global help for execute[USE:tabledelete.execute]
\help TableDelete.execute

#@ tabledelete.help
tabledelete.help('help')

#@ global ? for help[USE:tabledelete.help]
\? TableDelete.help

#@ global help for help[USE:tabledelete.help]
\help TableDelete.help

#@ tabledelete.limit
tabledelete.help('limit')

#@ global ? for limit[USE:tabledelete.limit]
\? TableDelete.limit

#@ global help for limit[USE:tabledelete.limit]
\help TableDelete.limit

#@ tabledelete.order_by
tabledelete.help('order_by')

#@ global ? for order_by[USE:tabledelete.order_by]
\? TableDelete.order_by

#@ global help for order_by[USE:tabledelete.order_by]
\help TableDelete.order_by

#@ tabledelete.where
tabledelete.help('where')

#@ global ? for where[USE:tabledelete.where]
\? TableDelete.where

#@ global help for where[USE:tabledelete.where]
\help TableDelete.where
